.github/CONTRIBUTING.md

```diff
@@ -1,31 +1,10 @@
 # Contribution guidelines
 
-  - **To add to the list**: submit a pull request
-  - **To remove from the list**: open an issue
-
-## I want to add a new entry to the list
-
-  - Contribute links to the packages/projects you are familiar with
-  - Search existed entry/suggestion before making a new one
-  - **Run specs (`crystal spec`) before opening a pull request**
-
-### List quality rules
-
-  - List items should be sorted alphabetically
-  - Each item should be limited to one link
-  - Description should be clear and concise. **Excessive information like *"... for Crystal programming language"* should be omitted**
-
-### New package requirements
-
-Package/project is a good candidate to add to the list if it is:
-
-  - Generally useful to the community
-  - Functional
-  - Actively maintained
-  - Stable
-  - Documented
-  - Has tests (when possible)
-
-## I want to open an issue
-
-Please open an issue if you find anything that could be enhanced or have any suggestions for making the list more useful.
+Keep in mind that the list is very mature by now, so the bar of getting something accepted is high. Only submit something unique and generally useful.
+
+  - **If you just created something, wait at least 30 days before submitting** to give it some time to mature and ensure it's not just a publish-and-forget type of project
+  - Actively maintained:
+    - Regular, recent commits
+    - Or, for finished projects, issues and pull requests are responded to
+  - Ensure your work follows general quality rules ([Crystal Shards 101](https://github.com/mccallofthewild/crystal-shards-101) if this is a Crystal shard)
+  - Run `crystal spec` before submitting a PR
```

Comment:
I assume this means in *this* repository although thats not clear at first glance.